### DISCUSSION TITLE
Legend/Estimated Printing Time Box Question

### DISCUSSION THREAD

**Discussion by djngnear** [supermerill/SuperSlicer#1880] [Q&A]
Is there a way to disable the Legend box in the GCode Preview from popping up every time the object is sliced? I know it's trivial but I always find myself closing it. Thanks

**Comment by supermerill** [comment]
Why not.
You close it with the `L` key?

creating the issue #1897 for this feature request.

**Comment by djngnear** [comment]
Well that helps! I didn’t know that thank you

——
Derek
